src/components/text-editor/examples/text-editor-as-form-component.tsx

```diff
@@ -1,20 +1,28 @@
 import { Component, h, State } from '@stencil/core';
 import { FormComponentFormData, schema } from './text-editor-form-data';
 /**
- * Basic example
+ * Using the text editor as a form component
  *
- * Here we have a simple form that uses the `limel-text-editor` component
+ * Here we have a simple form that uses the `limel-text-editor` component,
+ * instead of a regular text input field.
+ *
+ * :::note
+ * This allows the user to write rich text, with markdown support, in the form.
+ * But keep in mind that the value will be saved as a markdown string,
+ * and can also contain HTML tags, depending on what the users input
+ * in the filed.
+ * :::
  * @sourceFile text-editor-form-data.ts
  */
 @Component({
-    tag: 'limel-example-text-editor-basic',
+    tag: 'limel-example-text-editor-as-form-component',
     shadow: true,
 })
-export class TextEditorBasicExample {
+export class TextEditorAsFormComponentExample {
     @State()
     private formData: FormComponentFormData = {
-        name: 'Ali',
-        value: '<p>I am the greatest</p>',
+        name: 'Muhammad Ali',
+        value: '<p>I am the <b>greatest</b>.</p>',
```

Comment:
@LawrenceBorst @civing not related to the current PR, but shouldn't we be able to give a markdown as `value`? When I remove the `<p>` tags, and replace the `<b>` tags with `*`, I will get the string straightly here. Or do we need to do something else?

Reply:
Did you try with `I am the **greatest**.`? I tried it a while back